WebRoot/messages/ZmMsg.properties

```diff
@@ -1991,7 +1991,7 @@ localeName_sv = Swedish
 localeName_tr = Turkish
 localeName_th = Thai
 localeName_uk = Ukrainian
-
+localeName_ta = Tamil
```

Comment:
This change should go via login patch

Reply:
Ok, I am making this change in "zm-web-client-private" as well.  So that, it comes via login patch.